libs/block-scripts/shared/webpack-config.js

```diff
@@ -48,11 +48,17 @@ const generateBaseWebpackConfig = async (mode) => {
         ],
         "block entry point",
       ),
+      ...((await fs.pathExists("./src/example-graph.ts"))
+        ? {
+            "example-graph": "./src/example-graph.ts",
+          }
+        : {}),
```

Comment:
See https://github.com/blockprotocol/blockprotocol/pull/1223/files#r1149557972

If we want to require it we can simplify these lines